Content.Shared/Nuke/SharedNuke.cs

```diff
@@ -39,6 +39,13 @@ public enum NukeStatus : byte
         COOLDOWN
     }
 
+    public enum NukesAvailable : byte
+    {
+        ALL,
+        STATION,
+        SINDICAT
+    }
```

Comment:
The solution of having an Enum for explicit 'modes' of nuke tracking being 'All', 'Station', and 'Syndicate' is a very hardcoded solution to what should be entirely doable in YAML. I feel for this pr to make sense the Pinpointer itself just needs to be slightly reworked.